docs/elements.md

```diff
@@ -923,6 +923,7 @@ def fib(n):
 | `file-name`          | string  | -                  | The name of this file; will be used to store this file in the `_files` submitted answer                                                                                                                                                                                                                                                                                                                                                               |
 | `ace-mode`           | string  | None               | Specifies an Ace editor mode to enable things like intelligent code indenting and syntax highlighting; see the full list of modes [here](https://github.com/ajaxorg/ace/tree/master/lib/ace/mode).                                                                                                                                                                                                                                                    |
 | `ace-theme`          | string  | `ace/theme/chrome` | Specifies an Ace editor theme; see the full list of themes [here](https://github.com/ajaxorg/ace/tree/master/lib/ace/theme).                                                                                                                                                                                                                                                                                                                          |
+| `font-size`          | float   | None               | Specifies the font size (in px) for the Ace editor.                                                                                                                                                                                                                                                                                                                                                                                                   |
```

Comment:
It seems font size actually supports either a number represented as pixels or a string with a CSS-style size (so `1rem` or something similar would also be acceptable). Not sure if it's worth accepting this option or not, just bringing it up.

As for the default, it would probably bring more questions than information to keep the None here. Documentation is not very clear about the default, but the source code for ace seems to indicate its default as 12px.

Reply:
Sure, that makes sense.